manim/utils/color/core.py

```diff
@@ -215,9 +251,8 @@ def _internal_from_integer(value: int, alpha: float) -> ManimColorInternal:
             dtype=ManimColorDType,
         )
 
-    # TODO: Maybe make 8 nibble hex also convertible ?
     @staticmethod
-    def _internal_from_hex_string(hex: str, alpha: float) -> ManimColorInternal:
+    def _internal_from_hex_string(hex: str, alpha: float | None) -> ManimColorInternal:
```

Comment:
What does `alpha=None` represent? Is it the alpha for when `len(hex)==6` (in which case the doc string should be updated)?